app/tests/README.md

```diff
@@ -0,0 +1,19 @@
+# Houston tests
+
+This file contains some technical information for test files.
+
+### Overview
+
+All tests are expected to be ran in parallel and return a promise of an object in the form of:
+```javaScript
+{
+  error: ['error message strings'],
+  warning: ['warning message strings']
+}
+```
+
+NOTE: If you expect to save data to the database, use the `.update()` function instead of `.save()`. This will prevent data overwrites when running tests in parallel.
```

Comment:
That's mostly for the update functions like the `apphub.js` file. I didn't want to put the fetch and parse logic in the application model, and to keep the code dry, I moved it to the test files. Also, I didn't want to limit the possibilities down the road. The note was more for people that don't understand that the save function will overwrite the current application with the old application object in memory.

use cases:
`apphub.js` saves github label and price to database
`desktop.js` saves application's name
`icon.js` saves the icon